.circleci/continue_config.yml

```diff
@@ -66,7 +66,7 @@ executors:
     environment:
       MAVEN_OPTS: "-Xmx4G -XX:+ExitOnOutOfMemoryError -XX:CompressedClassSpaceSize=3g"
       MAVEN_INSTALL_OPTS: "-Xmx2G -XX:+ExitOnOutOfMemoryError -XX:CompressedClassSpaceSize=3g"
-      MAVEN_FAST_INSTALL: "-B -V --quiet -T C1 -DskipTests -Dair.check.skip-all -Dmaven.javadoc.skip=true"
```

Comment:
Just to make sure I understand, the format of this argument changed in between Maven versions?

Reply:
Yes, exactly. I think 1C (one core) always worked but C1 (core one) used to work and no longer does.